src/services/usersService.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const createId = require('../utils/createId');
+
+const users = [];
+
+function init() {
+  users.length = 0;
+}
+
+function get() {
+  return users;
+}
+
+function create(name) {
+  const user = {
+    id: createId(users),
+    name,
+  };
+
+  users.push(user);
+
+  return user;
+}
```

Comment:
The `create` function does not validate the `name` parameter. Consider adding validation to ensure that the `name` is a non-empty string and meets any other requirements before creating a user.